# =============================================================================
# Copyright (C) 2026 Qualcomm Technologies, Inc.
# All rights reserved.
# Confidential and Proprietary - Qualcomm Technologies, Inc.
# =============================================================================

diff --git a/Kconfig b/Kconfig
index 013175c..8aa24b1 100644
--- a/Kconfig
+++ b/Kconfig
@@ -1,6 +1,6 @@
 config RTL8188EU
 	tristate "Realtek 8188E USB WiFi"
 	depends on USB
-	---help---
+	help
 	  Help message of RTL8188EU
 
diff --git a/os_dep/linux/wifi_regd.c b/os_dep/linux/wifi_regd.c
index 81e1dc7..a11e21b 100644
--- a/os_dep/linux/wifi_regd.c
+++ b/os_dep/linux/wifi_regd.c
@@ -405,7 +405,7 @@ int rtw_regd_init(struct wiphy *wiphy)
 	wiphy->regulatory_flags &= ~REGULATORY_DISABLE_BEACON_HINTS;
 #endif
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 39))
 	wiphy->regulatory_flags |= REGULATORY_IGNORE_STALE_KICKOFF;
 #endif
 
